json-sports-team/styles.css

```diff
@@ -0,0 +1,56 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --dark-grey: #0a0a23;
+  --light-grey: #f5f6f7;
+  --white: #ffffff;
+  --black: #000;
+}
+
+body {
+  background-color: var(--dark-grey);
+  text-align: center;
```

Comment:
This project already look very good, and is mobile friendly. But I did notice that the text gets very close to the edges on smaller devices.

Adding 10 or 15 pixels of padding to the body can help with that.